### PR TITLE
chore(skip-release): Prepare for 1.17.1 (#779)

### LINKED ISSUE #779: chore: prepare release 1.17.1

### PR DESCRIPTION
**Pull request by adietish**
fixes #779

### CODE REVIEW

**Comment by adietish** [comment]
rebased

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -11,6 +11,10 @@
   ]]></description>
 
   <change-notes><![CDATA[
+    <p><b>1.18.0</b></p>
```
Comment (jeffmaury) [inline src/main/resources/META-INF/plugin.xml:14]: Not align with previous file

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by adietish** [comment]
1.17.1 was published in the marketplace: https://plugins.jetbrains.com/plugin/13234-quarkus-tools/versions/stable/318968
image

**Pull request merged**